python/lsst/skymap/tractInfo.py

```diff
@@ -185,16 +224,25 @@ def findPatch(self, coord):
         return self.getPatchInfo(patchInd)
 
     def findPatchList(self, coordList):
-        """Find patches containing the specified list of coords
+        """Find patches containing the specified list of coords.
+
+        Parameters
+        ----------
+        coordList : `list` of `lsst.afw.geom.SpherePoint`
+            ICRS sky coordinates to search for.
 
-        @param[in] coordList: list of sky coordinates (lsst.afw.geom.SpherePoint)
-        @return list of PatchInfo for patches that contain, or may contain, the specified region.
+        Returns
+        -------
+        result : `list` of `lsst.skymap.PatchInfo`
+            List of PatchInfo for patches that contain, or may contain, the specified region.
             The list will be empty if there is no overlap.
 
-        @warning:
-        * This may give incorrect answers on regions that are larger than a tract
-        * This uses a naive algorithm that may find some patches that do not overlap the region
-            (especially if the region is not a rectangle aligned along patch x,y).
+        Notes
+        -----
+        warning:
+        - This may give incorrect answers on regions that are larger than a tract
```

Comment:
Please fix to make warning bold and render the items as an unordered list:
```
**warning:**

- This may give...
- This uses...
  (especially if..
```